config/default/common/config/metadata/layers/gedi/GEDI_ISS_L3_Canopy_Height_StdDev_RH100_201904-202108.md

```diff
@@ -0,0 +1,3 @@
+The Canopy Height Standard Deviation (L3, v2, 2019 APR to 2021 AUG) layer is the standard deviation of RH100 for all valid footprints per 1 km x 1 km grid cell. RH100 or relative height is the 100th percentile of waveform energy relative to ground elevation. This layer is from the version 2 release of the Global Elevation Dynamics Investigation (GEDI) L3 gridded land surface metrics data covering the period between the 19th and the 138th mission weeks (2019-04-19 to 2021-08-04). It is derived from the GEDI L2B version 2 geolocated laser footprints metric data product. The imagery resolution is 1 km and is available globally within -52 and 52 degrees latitude.
```

Comment:
Should these dates: (2019-04-19 to 2021-08-04) be in the same format as other dates throughout the app? e.g. 2019 APR 19 to 2021 AUG 04?

Also... should the be using a monospace font?

Reply:
Ok but also the description here has two different date formats: "2019 APR to 2021 AUG" and "2019-04-19 to 2021-08-04", is that intentional?

Reply:
yes, unfortunately :(

Reply:
Should all be in the correct date format now!

Reply:
Looks great!